packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -68,9 +68,12 @@ class _ToolbarContainerLayout extends SingleChildLayoutDelegate {
 /// menu").
 ///
 /// App bars are typically used in the [Scaffold.appBar] property, which places
-/// the app bar as a fixed-height widget at the top of the screen. For a
-/// scrollable app bar, see [SliverAppBar], which embeds an [AppBar] in a sliver
-/// for use in a [CustomScrollView].
+/// the app bar as a fixed-height widget at the top of the screen. [Scaffold] also
+/// removes the bottom padding of the [MediaQuery] the app bar inherits from. When used
```

Comment:
Although it's true that the AppBar is wrapped in a MediaQuery that zeroes the bottom padding, it seems odd to mention it, since the bottom padding shouldn't matter to most AppBar users. I understand that this addition is intended to address #27848; maybe what's needed is a separate paragraph about heroic appbars avoiding notches etc.